app/controllers/v0/user_action_events_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module V0
+  class UserActionEventsController < ApplicationController
+    service_tag 'identity'
+
+    def index
+      user_verification = current_user.user_verification
+
+      start_date = params[:start_date].present? ? Date.parse(params[:start_date]) : 1.month.ago.to_date
+      end_date = params[:end_date].present? ? Date.parse(params[:end_date]).end_of_day : Time.zone.now
+      page = (params[:page].presence || 1).to_i
+      per_page = (params[:per_page].presence || 10).to_i
+
+      user_actions = UserAction
+                     .where(subject_user_verification_id: user_verification.id)
```

Comment:
are you able to do `.where(subject_user_verification: user_verification)`, just to make it a tiny bit more 'Rails-like'. Also, I don't see `user_verification` used anywhere else, so might as well just do `current_user.user_verification` here